Donation-platform/src/main/java/com/google/CharitiesAndTags.java

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Google LLC
+//
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public final class CharitiesAndTags {
```

Comment:
Why are there two of the same class?

Reply:
I think it's because in my first commit, I'd named the file Charities.java, but then I had to rename the file to be the same as the class name, so I re-committed with CharitiesAndTags.java.

Reply:
Ok, I see. Maybe we should re-name the class to HardcodedCharitiesAndTags or ExampleCharitiesAndTags? Otherwise LGTM.

Reply:
+1 please rename.

Reply:
renamed